db/migrate/20220617151846_rename_default_categories_regular_setting.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class RenameDefaultCategoriesRegularSetting < ActiveRecord::Migration[7.0]
```

Comment:
Shouldn't this be a post-migration?

Reply:
Yes, for sure! I've moved it to `db/post_migrate`.